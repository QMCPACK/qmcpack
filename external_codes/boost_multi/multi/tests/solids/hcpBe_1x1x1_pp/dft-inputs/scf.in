&CONTROL
   calculation     = 'scf'
   disk_io         = 'low'
   outdir          = 'pwscf_output'
   prefix          = 'pwscf'
   pseudo_dir      = './'
   restart_mode    = 'from_scratch'
   tprnfor         = .false.
   tstress         = .false.
   verbosity       = 'high'
   wf_collect      = .true.
/

&SYSTEM
   celldm(1)       = 1.0
   degauss         = 0.0001
   ecutrho         = 1000
   ecutwfc         = 250
   ibrav           = 0
   input_dft       = 'lda'
   nat             = 1
   nosym           = .true.
   ntyp            = 1
   occupations     = 'smearing'
   smearing        = 'fermi-dirac'
   tot_charge      = 0
/

&ELECTRONS
   conv_thr        = 1e-08
   electron_maxstep = 1000
   mixing_beta     = 0.7
/


ATOMIC_SPECIES 
   Be 9.0122 Be.BFD.upf

ATOMIC_POSITIONS alat
   Be       0.00000000       0.00000000       0.00000000 

K_POINTS automatic
   1 1 1  0 0 0 

CELL_PARAMETERS cubic
         4.32747284       0.00000000       0.00000000 
        -2.16373642       3.74770142       0.00000000 
         0.00000000      -0.00000000       6.78114995 


