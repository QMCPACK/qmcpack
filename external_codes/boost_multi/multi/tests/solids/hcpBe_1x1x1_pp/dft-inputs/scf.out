
     Program PWSCF v.4.2        starts on 31Oct2014 at 13:24:28 

     This program is part of the open-source Quantum ESPRESSO suite
     for quantum simulation of materials; please acknowledge
         "P. Giannozzi et al., J. Phys.:Condens. Matter 21 395502 (2009);
          URL http://www.quantum-espresso.org", 
     in publications or presentations arising from this work. More details at
     http://www.quantum-espresso.org/wiki/index.php/Citing_Quantum-ESPRESSO

     Parallel version (MPI), running on    16 processors
     R & G space division:  proc/pool =   16

     EXPERIMENTAL VERSION WITH EXACT EXCHANGE

     Current dimensions of program PWSCF are:
     Max number of different atomic species (ntypx) = 10
     Max number of k-points (npk) =  40000
     Max angular momentum in pseudopotentials (lmaxx) =  3
     Waiting for input...

     XC functional enforced from input :
     Exchange-correlation      = LDA (1100)
     EXX-fraction              =        0.00
     !!! Any further DFT definition will be discarded
     !!! Please, verify this is what you really want !

     file Be.BFD.upf: wavefunction(s)  0s 0p renormalized

     Subspace diagonalization in iterative solution of the eigenvalue problem:
     Too few electrons for parallel  algorithm: we need # of bands >= SQRT(nproc)
     a serial algorithm will be used


     Planes per process (thick) : nr3 =  72 npp =    5 ncplane =  2025
 
     Proc/  planes cols     G    planes cols    G      columns  G
     Pool       (dense grid)       (smooth grid)      (wavefct grid)
        1     5     80     3672    5     80     3672     19      455
        2     5     80     3672    5     80     3672     19      455
        3     5     80     3672    5     80     3672     19      455
        4     5     80     3672    5     80     3672     19      455
        5     5     80     3672    5     80     3672     19      455
        6     5     80     3672    5     80     3672     19      455
        7     5     81     3677    5     81     3677     19      455
        8     5     81     3677    5     81     3677     20      462
        9     4     80     3670    4     80     3670     20      462
       10     4     80     3670    4     80     3670     20      462
       11     4     80     3670    4     80     3670     20      462
       12     4     80     3670    4     80     3670     20      462
       13     4     80     3670    4     80     3670     20      462
       14     4     81     3675    4     81     3675     20      462
       15     4     81     3675    4     81     3675     20      462
       16     4     81     3675    4     81     3675     20      462
     tot     72   1285    58761   72   1285    58761    313     7343
 


     bravais-lattice index     =            0
     lattice parameter (a_0)   =       1.0000  a.u.
     unit-cell volume          =     109.9772 (a.u.)^3
     number of atoms/cell      =            1
     number of atomic types    =            1
     number of electrons       =         2.00
     number of Kohn-Sham states=            5
     kinetic-energy cutoff     =     250.0000  Ry
     charge density cutoff     =    1000.0000  Ry
     convergence threshold     =      1.0E-08
     mixing beta               =       0.7000
     number of iterations used =            8  plain     mixing
     Exchange-correlation      = LDA (1100)
     EXX-fraction              =        0.00

     celldm(1)=   1.000000  celldm(2)=   0.000000  celldm(3)=   0.000000
     celldm(4)=   0.000000  celldm(5)=   0.000000  celldm(6)=   0.000000

     crystal axes: (cart. coord. in units of a_0)
               a(1) = (  4.327473  0.000000  0.000000 )  
               a(2) = ( -2.163736  3.747701  0.000000 )  
               a(3) = (  0.000000  0.000000  6.781150 )  

     reciprocal axes: (cart. coord. in units 2 pi/a_0)
               b(1) = (  0.231082  0.133415  0.000000 )  
               b(2) = (  0.000000  0.266830  0.000000 )  
               b(3) = (  0.000000  0.000000  0.147468 )  


     PseudoPot. # 1 for Be read from file Be.BFD.upf
     Pseudo is Norm-conserving, Zval =  2.0
     Generated by new atomic code, or converted to UPF format
     Using radial grid of 2000 points,  1 beta functions with: 
                l(1) =   0

     atomic species   valence    mass     pseudopotential
        Be             2.00     9.01220     Be( 1.00)

     No symmetry found
                                    s                        frac. trans.

      isym =  1     identity                                     

 cryst.   s( 1) = (     1          0          0      )    f =(  0.0000000 )
                  (     0          1          0      )       (  0.0000000 )
                  (     0          0          1      )       (  0.0000000 )

 cart.    s( 1) = (  1.0000000  0.0000000  0.0000000 )    f =(  0.0000000 )
                  (  0.0000000  1.0000000  0.0000000 )       (  0.0000000 )
                  (  0.0000000  0.0000000  1.0000000 )       (  0.0000000 )


     point group C_1 (1)    
     there are  1 classes
     the character table:

       E    
A      1.00

     the symmetry operations in each class:
     E        1

   Cartesian axes

     site n.     atom                  positions (a_0 units)
         1           Be  tau(  1) = (   0.0000000   0.0000000   0.0000000  )

   Crystallographic axes

     site n.     atom                  positions (cryst. coord.)
         1           Be  tau(  1) = (  0.0000000  0.0000000  0.0000000  )

     number of k points=     1  gaussian broad. (Ry)=  0.0001     ngauss = -99
                       cart. coord. in units 2pi/a_0
        k(    1) = (   0.0000000   0.0000000   0.0000000), wk =   2.0000000

                       cryst. coord.
        k(    1) = (   0.0000000   0.0000000   0.0000000), wk =   2.0000000

     G cutoff =   25.3303  (  58761 G-vectors)     FFT grid: ( 45, 45, 72)

     Largest allocated arrays     est. size (Mb)     dimensions
        Kohn-Sham Wavefunctions         0.03 Mb     (    455,   5)
        NL pseudopotentials             0.01 Mb     (    455,   1)
        Each V/rho on FFT grid          0.15 Mb     (  10125)
        Each G-vector array             0.03 Mb     (   3672)
        G-vector shells                 0.01 Mb     (   1676)
     Largest temporary arrays     est. size (Mb)     dimensions
        Auxiliary wavefunctions         0.14 Mb     (    455,  20)
        Each subspace H/S matrix        0.01 Mb     (     20,  20)
        Each <psi_i|beta_j> matrix      0.00 Mb     (      1,   5)
        Arrays for rho mixing           1.24 Mb     (  10125,   8)

     Initial potential from superposition of free atoms

     starting charge    0.15914, renormalised to    2.00000
     Starting wfc are    4 atomic +    1 random wfc

     total cpu time spent up to now is      0.14 secs

     per-process dynamical memory:     2.8 Mb

     Self-consistent Calculation

     iteration #  1     ecut=   250.00 Ry     beta=0.70
     Davidson diagonalization with overlap
     ethr =  1.00E-02,  avg # of iterations =  3.0

     Threshold (ethr) on eigenvalues was too large:
     Diagonalizing with lowered threshold

     Davidson diagonalization with overlap
     ethr =  3.03E-04,  avg # of iterations =  1.0

     total cpu time spent up to now is      0.18 secs

     total energy              =      -2.79696832 Ry
     Harris-Foulkes estimate   =      -2.79847747 Ry
     estimated scf accuracy    <       0.00628819 Ry

     iteration #  2     ecut=   250.00 Ry     beta=0.70
     Davidson diagonalization with overlap
     ethr =  3.14E-04,  avg # of iterations =  1.0

     total cpu time spent up to now is      0.20 secs

     total energy              =      -2.79748008 Ry
     Harris-Foulkes estimate   =      -2.79743803 Ry
     estimated scf accuracy    <       0.00024829 Ry

     iteration #  3     ecut=   250.00 Ry     beta=0.70
     Davidson diagonalization with overlap
     ethr =  1.24E-05,  avg # of iterations =  1.0

     total cpu time spent up to now is      0.22 secs

     total energy              =      -2.79748496 Ry
     Harris-Foulkes estimate   =      -2.79748143 Ry
     estimated scf accuracy    <       0.00001506 Ry

     iteration #  4     ecut=   250.00 Ry     beta=0.70
     Davidson diagonalization with overlap
     ethr =  7.53E-07,  avg # of iterations =  1.0

     total cpu time spent up to now is      0.23 secs

     End of self-consistent calculation

          k = 0.0000 0.0000 0.0000 (  7343 PWs)   bands (ev):

    -5.5792   3.3453   8.2300  19.9533  19.9533

     occupation numbers 
     2.0000   0.0000   0.0000   0.0000   0.0000

     the Fermi energy is     0.8026 ev

!    total energy              =      -2.79748507 Ry
     Harris-Foulkes estimate   =      -2.79748507 Ry
     estimated scf accuracy    <          1.1E-09 Ry

     The total energy is the sum of the following terms:

     one-electron contribution =       0.20801408 Ry
     hartree contribution      =       0.13650850 Ry
     xc contribution           =      -0.99898481 Ry
     ewald contribution        =      -2.14302284 Ry
     - averaged Fock potential =       0.00000000 Ry
     + Fock energy             =       0.00000000 Ry
     smearing contrib. (-TS)   =       0.00000000 Ry

     convergence has been achieved in   4 iterations

     Writing output data file pwscf.save

     Writing output data file pwscf.save
 
     init_run     :      0.11s CPU      0.12s WALL (       1 calls)
     electrons    :      0.08s CPU      0.08s WALL (       1 calls)

     Called by init_run:
     wfcinit      :      0.00s CPU      0.00s WALL (       1 calls)
     potinit      :      0.04s CPU      0.04s WALL (       1 calls)

     Called by electrons:
     c_bands      :      0.03s CPU      0.03s WALL (       5 calls)
     sum_band     :      0.01s CPU      0.01s WALL (       5 calls)
     v_of_rho     :      0.00s CPU      0.01s WALL (       5 calls)
     v_h          :      0.00s CPU      0.00s WALL (       5 calls)
     v_xc         :      0.00s CPU      0.00s WALL (       5 calls)
     mix_rho      :      0.00s CPU      0.00s WALL (       5 calls)

     Called by c_bands:
     init_us_2    :      0.00s CPU      0.00s WALL (      11 calls)
     cegterg      :      0.03s CPU      0.03s WALL (       5 calls)

     Called by sum_band:

     Called by *egterg:
     h_psi        :      0.02s CPU      0.03s WALL (      13 calls)
     g_psi        :      0.00s CPU      0.00s WALL (       7 calls)
     cdiaghg      :      0.01s CPU      0.00s WALL (      11 calls)
     cegterg:over :      0.00s CPU      0.00s WALL (       7 calls)
     cegterg:upda :      0.00s CPU      0.00s WALL (       7 calls)
     cegterg:last :      0.00s CPU      0.00s WALL (       5 calls)

     Called by h_psi:
     h_psi:vloc   :      0.02s CPU      0.03s WALL (      13 calls)
     h_psi:vnl    :      0.00s CPU      0.00s WALL (      13 calls)
     add_vuspsi   :      0.00s CPU      0.00s WALL (      13 calls)

     General routines
     calbec       :      0.00s CPU      0.00s WALL (      13 calls)
     cft3s        :      0.03s CPU      0.04s WALL (     170 calls)
 
     Parallel routines
     fft_scatter  :      0.02s CPU      0.01s WALL (     170 calls)
     EXX routines
 
     PWSCF        :     0.30s CPU time,        0.35s WALL time

 
   This run was terminated on:  13:24:28  31Oct2014            

=------------------------------------------------------------------------------=
   JOB DONE.
=------------------------------------------------------------------------------=
